inst/extdata/database/tedfs/Tech/Haber-Bosch with ASU.csv

```diff
@@ -13,3 +17,7 @@ Input|Electricity,Output|Ammonia,2014,Synthesis process,0.4,,MWh,1.0,t,,Morgan14
 Input|Electricity,Output|Ammonia,2018,#,0.64,,MWh,1.0,t,,Ikaheimo18,Page 4
 Input|Electricity,Output|Ammonia,2017,Compressors,5.2,1.3,GJ,1.0,t,,Bazzanella17,"Page 56, Section 4.2.2"
 Input|Electricity,Output|Ammonia,2017,Air-separation unit,0.33,,MWh,1.0,t,,Bazzanella17,"Page 57, Section 4.2.3"
+Input|Electricity,Output|Ammonia,,H2 compression and others,0.063,,kWh,1,kWh;LHV,Found in supplementary information,Stolz22,Table 5
+Input|Nitrogen,Output|Ammonia,,#,0.159,,kg,1,kWh;LHV,Found in supplementary information,Stolz22,Table 5
+Input|Electricity,Input|Nitrogen,,Air-separation unit,0.162,,kWh,1,kg,"NOT WORKING. Want to add the elec required from ASU, but can't relate this line to the one above (nitrogen to ammonia). Data found in supplementary information",Stolz22,Table 5
```

Comment:
There would be two options here:

- Split off the ASU and make it a separate process. — I have a slight preference against that, because that green ammonia plants running on hydrogen (and not fossil gas) will always require an integrated ASU in future.
- Update that line such that Nitrogen is not an input to this process. Since the ASU for providing the nitrogen is integrated, the plant as a whole has not Nitrogen Input.

Does that make sense?

Reply:
Found an easier alternative solution (that I'll add in the next PR). Simply exchanging line 21 and 22 yield the expected behaviour, and calculates the electricity requirement per ton ammonia.